geotrellis/geotrellis-site
# Adds OTM Tree Prioritization demo and misc fixes

# Overview
OTM does not use GeoTrellis. There were two options described for resolving the issue in the original issue: replace the project with the Tree Prioritization demo, or leave the project as-is and add the Tree Prioritization demo to the Demos section. I went with the latter option.

While in there, addressed #102 & #103. I also noticed that we are linking to a different Building Footprint demo than the screenshot/description on the homepage, as of #105. The screenshot/description have been updated to reflect this change.

Closes #100, closes #102, closes #103

# Demo
<img width="1792" alt="Screen Shot 2019-11-01 at 10 01 13 AM" src="https://user-images.githubusercontent.com/5672295/68014124-f5bef980-fc8e-11e9-8999-d96780ddf480.png">
<img width="241" alt="Screen Shot 2019-11-01 at 10 17 00 AM" src="https://user-images.githubusercontent.com/5672295/68014849-eb056400-fc90-11e9-8ace-1bb15a748d63.png">

# Testing
* `git pull`
* Scroll down to the demos section of homepage, there should be 3 demos now.
* Confirm that all links work / go to the right place.
* Confirm that hovering on the Geotrellis logo no longer shows "Azavea Careers homepage" text
* Confirm that CTRL+click or CMD+click opens project cards in a new tab
